task/aws/resources/mocks/gen.go

```diff
@@ -1,5 +0,0 @@
-package mocks
```

Comment:
By moving to using `rclone` to validate bucket access, these mocks are no longer useful. I'll probably revisit the idea at some point.